docs/root/configuration/network_filters/tcp_proxy_filter.rst

```diff
@@ -22,8 +22,10 @@ statistics are rooted at *tcp.<stat_prefix>.* with the following statistics:
 
   downstream_cx_total, Counter, Total number of connections handled by the filter
   downstream_cx_no_route, Counter, Number of connections for which no matching route was found or the cluster for the route was not found
+  downstream_cx_tx_bytes, Histogram, Bytes written to the downstream connection
```

Comment:
Nit: Should say "Size of bytes written" (and "Size of bytes read" below) to match the others.